src/tui.rs

```diff
@@ -182,7 +182,7 @@ impl Tui {
         crossterm::terminal::enable_raw_mode()?;
         crossterm::execute!(io(), EnterAlternateScreen, cursor::Hide)?;
         if self.mouse {
-            crossterm::execute!(io(), EnableMouseCapture)?;
+            crossterm::execute!(io())?;
```

Comment:
@BapRx Thank you, but I found this change break mouse scrolling feature on history view.
I'll investigate the problem too.

Reply:
Oh.. good catch, I haven't used history mode yet